components/AppBar.tsx

```diff
@@ -3,17 +3,15 @@ import Banner from "./Banner";
 
 const AppBar = (props: PropsWithChildren<{
 	position: 'absolute' | 'fixed' | 'relative' | 'static' | 'sticky',
-	offset: boolean,
 	className?: string,
 	backdropClassName?: string,
 }>) => {
 	const headerRef = useRef<HTMLDivElement>(null);
-	const appbarRef = useRef<HTMLDivElement>(null);
 	const offsetRef = useRef<HTMLDivElement>(null);
 
 	useEffect(() => {
 		if (headerRef.current && offsetRef.current) {
-			const offsetHeight = headerRef.current.clientHeight - ((!props.offset && appbarRef.current) ? appbarRef.current.clientHeight : 0);
+			const offsetHeight = headerRef.current.clientHeight;
```

Comment:
is this change relevant for this PR?

Reply:
it wasn't. This happened as main was not merged into this branch, not an issue now